core/notifications/src/email_executor/smtp/mod.rs

```diff
@@ -41,6 +41,7 @@ impl SmtpClient {
             .from(Mailbox::new(None, self.from_email.parse()?))
```

Comment:
Note: we can pass in a name here that should be displayed instead of the complete email.